File: 백준/Bronze/10809. 알파벳 찾기/알파벳 찾기.cc
#include <iostream>

using namespace std;

int main(void) {
    char word[101];
    char* p = word;
    int alphabet[26], index = 0;

    cin >> word;

    for (int i = 0; i < 26; i++) {
        alphabet[i] = -1;
    }

    while (*p != '\0') {
        if (alphabet[*p - 'a'] == -1) {
            alphabet[*p - 'a'] = index++;
        } else {
            index++;
        }
        p++;
    }

    for (int i = 0; i < 26; i++) {
        cout << alphabet[i] << " ";
    }
    cout << "\n";
    return 0;
}

File: 백준/Bronze/10809. 알파벳 찾기/알파벳 찾기.py
alphabet = [-1 for _ in range(26)]

word = input()
for i in range(len(word)) :
    alp_idx = ord(word[i]) - ord('a')
    if (alphabet[alp_idx] == -1) :
        alphabet[alp_idx] = i
        
for i in alphabet :
    print(i, end = ' ')